inspire_disambiguation/api.py

```diff
@@ -95,23 +98,44 @@ def cluster(ethnicity_model_path, distance_model_path, n_jobs, signature_block=N
         signature_block (str): Signature block indicating which block should be
             clustered. If set to None, clustering will run on all blocks.
     """
-    LOGGER.info("Pulling signatures for block '%s' from ES", signature_block)
-
+    LOGGER.info("Pulling signatures from ES", signature_block=signature_block)
+    _start = datetime.now()
     signatures = get_signatures(signature_block=signature_block)
     input_clusters = get_input_clusters(signatures)
-    LOGGER.debug(
-        "Got %s signature_blocks and %s input_clusters",
-        len(signatures),
-        len(input_clusters),
+    LOGGER.info(
+        "Input data",
+        signature_block=signature_block,
+        signatures_count=len(signatures),
+        curated_signatures_count=len(
+            [sig for sig in signatures if sig.get("is_curated_author_id")]
+        ),
+        input_clusters_count=len(input_clusters),
+        input_clusters=input_clusters,
     )
     distance_estimator = DistanceEstimator.get(
         ethnicity_model_path, distance_model_path
     )
 
     clusterer = Clusterer(distance_estimator)
     clusterer.load_data(signatures, input_clusters)
-    LOGGER.info("Starting clustering")
+    LOGGER.info(
+        "Signatures load time",
+        signature_block=signature_block,
+        time_took=datetime.now() - _start,
```

Comment:
I think this is a bit misleading since we are not only pulling/loading signatures from ES, we are also preparing the input_clusters, loading the publications and so on.

Reply:
:+1: